test/controllers/ideas_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class IdeasControllerTest < ActionController::TestCase
+  setup do
+    @idea = ideas(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:ideas)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create idea" do
+    assert_difference('Idea.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.